storage/rocksdb/ha_rocksdb.cc

```diff
@@ -7193,6 +7193,10 @@ static void move_wals_to_target_dir() {
         (my_errno() != EEXIST))
       rdb_fatal_error("Failed to create %s", rocksdb_wal_dir);
 
+    if ((my_mkdir(rocksdb_datadir, S_IRWXU, MYF(0)) == -1) &&
```

Comment:
What does "rebuilding a db" mean here? Specifically, is a testcase possible?

Reply:
I think it means When it try to bootstrap a new mysql instance.
you can repro it when rocksdb_wal_dir and rocksdb_data_dir point to different directory, and then try to bootstrap it.

Reply:
Test case added.